src/main/java/edu/wisc/my/rssToJson/service/RsstoJsonServiceImpl.java

```diff
@@ -51,6 +54,37 @@ public JSONObject getJsonFromURL(String endpoint) {
         }
         jsonToReturn.put("items", entries);
         jsonToReturn.put("status", "ok");
-        return jsonToReturn;
+        return getJSONCleaned(jsonToReturn);
     }
+
+    private JSONObject getJSONCleaned(JSONObject jsonToClean) {
+      String tellTaleHighBitIndicator = "\\u";
+      if (!jsonToClean.toString().contains(tellTaleHighBitIndicator)) {
+        return jsonToClean;
+      }
+      try {
+        Resource resource = new ClassPathResource("stringCleaner.json");
```

Comment:
There are two known problematic characters. Is that worth externalizing into the JSON file or would it be clearer to inline that bit of configuration in the Java?

This `try` block becomes ~2 lines of code with that inlining? So it's a tradeoff maybe worth making.